packages/tizen_notification/tizen/src/tizen_notification_plugin.cc

```diff
@@ -40,478 +43,389 @@ class TizenNotificationPlugin : public flutter::Plugin {
 
  private:
   template <typename T>
-  bool GetValueFromArgs(const flutter::EncodableValue *args, const char *key,
-                        T &out) {
-    if (std::holds_alternative<flutter::EncodableMap>(*args)) {
-      flutter::EncodableMap map = std::get<flutter::EncodableMap>(*args);
-      if (map.find(flutter::EncodableValue(key)) != map.end()) {
-        flutter::EncodableValue value = map[flutter::EncodableValue(key)];
-        if (std::holds_alternative<T>(value)) {
-          out = std::get<T>(value);
-          return true;
-        }
-      }
-    }
-    return false;
-  }
-
-  bool GetEncodableValueFromArgs(const flutter::EncodableValue *args,
-                                 const char *key,
-                                 flutter::EncodableValue &out) {
-    if (std::holds_alternative<flutter::EncodableMap>(*args)) {
-      flutter::EncodableMap map = std::get<flutter::EncodableMap>(*args);
-      if (map.find(flutter::EncodableValue(key)) != map.end()) {
-        out = map[flutter::EncodableValue(key)];
+  static bool GetValueFromEncodableMap(const flutter::EncodableMap *map,
+                                       const char *key, T &out) {
```

Comment:
We usually declare this function globally with an anonymous namespace.
How about moving?

https://github.com/flutter-tizen/plugins/issues/354 (Enclose the plugin class within an anonymous namespace)

Reply:
Oh, I just forgot. Thank you for reminding.